commoncode/ama-bot
# [CCP-120] Set Up Backend Server

### Task

[JIRA ticket](https://commoncode.atlassian.net/browse/CCP-120).

### Description

Initialised `package.json`, created entry file `server.js`, added doc of setting up environment on local.

### Steps to Test or Reproduce

1. Follow the doc to set up environment.
2. Run server and check if there's any weird message in command line.
